### ISSUE TITLE
OP_REQUIRES failed at save_restore_v2_ops.cc:184 : Not found: Key Cycle_DVF/conv10/BatchNorm/beta not found in checkpoint，pretrained_model is wrong

### ISSUE THREAD

**Issue by hhhhhumengshun** [alex04072000/CyclicGen#5]

**Comment by alex04072000** [comment]
The loaded pre-trained model need to match the imported network in the python code. For example, if you have `from CyclicGen_model_large import Voxel_flow_model` in your testing code. You need to load checkpoint file from `ckpt\CyclicGen_large\model`.
Could you please check whether the imported network and loaded ckpt are matched?

**Comment by hhhhhumengshun** [comment]
![out](https://user-images.githubusercontent.com/49120968/58608098-99693f80-82d4-11e9-9743-0315b34b3bc2.png)
but，I use pretrained model to get this image. the image is worse than your output in paper

**Comment by alex04072000** [comment]
I use the large model `(from CyclicGen_model_large import Voxel_flow_model)`, and load the ckpt of large model (ckpt/CyclicGen_large/model), the result is quite promising. Could you please check the model you're using? For input frames that contain large motions, we recommend to use the large model.
![out](https://user-images.githubusercontent.com/8827488/58610683-ee11b800-82de-11e9-851f-a0a96d725cd5.png)

**Comment by hhhhhumengshun** [comment]
can you send your pretrained model my  e-mail,thanks


 



------------------ 原始邮件 ------------------
发件人: "Yu-Lun Liu"<notifications@github.com>; 
发送时间: 2019年5月30日(星期四) 中午1:29
收件人: "alex04072000/CyclicGen"<CyclicGen@noreply.github.com>; 
抄送: "1914861332"<1914861332@qq.com>; "Author"<author@noreply.github.com>; 
主题: Re: [alex04072000/CyclicGen] OP_REQUIRES failed at save_restore_v2_ops.cc:184 : Not found: Key Cycle_DVF/conv10/BatchNorm/beta not found in checkpoint，pretrained_model is wrong (#5)




I use the large model (from CyclicGen_model_large import Voxel_flow_model), and load the ckpt of large model (ckpt/CyclicGen_large/model), the result is quite promising. Could you please check the model you're using? For input frames that contain large motions, we recommend to use the large model.

**Comment by alex04072000** [comment]
The pre-trained model can be downloaded [here](https://drive.google.com/open?id=1X7PWDY2nAx8ZeSLso5qeypRUCDokNFms) as described in readme.

**Comment by treammm** [comment]
嚶嚶...

**Comment by hhhhhumengshun** [comment]
I download this pretrained model. but I use 640*480 image,but the result is worse than yours,can you give me your model? thanks 




------------------ 原始邮件 ------------------
发件人: "Yu-Lun Liu"<notifications@github.com>;
发送时间: 2019年5月30日(星期四) 晚上8:01
收件人: "alex04072000/CyclicGen"<CyclicGen@noreply.github.com>;
抄送: "1914861332"<1914861332@qq.com>;"Author"<author@noreply.github.com>;
主题: Re: [alex04072000/CyclicGen] OP_REQUIRES failed at save_restore_v2_ops.cc:184 : Not found: Key Cycle_DVF/conv10/BatchNorm/beta not found in checkpoint，pretrained_model is wrong (#5)




The pre-trained model can be downloaded here as described in readme.

**Comment by alex04072000** [comment]
I use exactly the same ckpt as described in readme.
I tested this middlebury sequence, while using frame07.png and frame09.png as inputs, the output is poor as your result due to the large motion. But using frame07.png and frame08.png as inputs, the result is correct. I think this is because the receptive field is still too small to handle such a large motion. 
![out2](https://user-images.githubusercontent.com/8827488/58703695-103d3000-83dc-11e9-8355-4660fd844000.png)

**Comment by hhhhhumengshun** [comment]
yes, i use frame09 and frame11,if the size is 384*512. the result is good .but the size is 480*640. the result is poor .the ball optical flow is poor.



------------------ 原始邮件 ------------------
发件人: "Yu-Lun Liu"<notifications@github.com>;
发送时间: 2019年5月31日(星期五) 晚上7:41
收件人: "alex04072000/CyclicGen"<CyclicGen@noreply.github.com>;
抄送: "1914861332"<1914861332@qq.com>;"Author"<author@noreply.github.com>;
主题: Re: [alex04072000/CyclicGen] OP_REQUIRES failed at save_restore_v2_ops.cc:184 : Not found: Key Cycle_DVF/conv10/BatchNorm/beta not found in checkpoint，pretrained_model is wrong (#5)




I use exactly the same ckpt as described in readme.
 I tested this middlebury sequence, while using frame07.png and frame09.png as inputs, the output is poor as your result due to the large motion. But using frame07.png and frame08.png as inputs, the result is correct. I think this is because the receptive field is still too small to handle such a large motion.

**Comment by alex04072000** [comment]
Yes, the larger the image resolution and temporal gap between input frames, the larger the motions. And this causes the network predicting wrong flow maps. So we intend to use a slightly larger model than DVF to overcome such problems. In this case, large model still performs better than original-sized model(DVF). However, with 640x480 resolution and time interval of 2 frames, the motions are still to large for the large model to handle.
